src/plugins/intel_cpu/src/nodes/conv.cpp

```diff
@@ -1250,7 +1250,7 @@ void MKLDNNConvolutionNode::prepareParams() {
     execPtr = nullptr;
     auto cache = getRuntimeCache();
     auto result = cache->getOrCreate(key, builder);
-
+    VERBOSE_HELPER_NODE_PREPARE_PARAMS(result.second);
```

Comment:
cache hit / miss is seemed to be one of the main part of prepareParams logic.
Can we instead return HIT / MISS as a prepareParams result (maybe even as element of some structure, in case we will need more things to return).
Then we will have to insert this macro only in base class, where prepareParams is called.
The downside is this return value will be ignored for non-DEBUG_CAPS builds

Reply:
Better to be discussed

Reply:
@maxnick @dmitry-gorokhov
Could you please comment on this